tools/python/upload_and_run_browserstack_tests.py

```diff
@@ -0,0 +1,93 @@
+import argparse
+import json
+import time
+
+import requests
+
+parser = argparse.ArgumentParser("Upload and run BrowserStack tests")
+
+parser.add_argument("--id", type=str, help="BrowserStack user ID")
+parser.add_argument("--token", type=str, help="BrowserStack token")
```

Comment:
is this value secret?

passing it on the command line might be fine in the context of a CI pipeline where it is redacted from logs, but command line arguments are generally not hidden. e.g., when running locally, the command line arguments will be visible to other processes and also in the command history.

some alternatives are to pass the secret value via stdin or in an environment variable.

if we don't expect to run this script much locally, this is less of a concern.

Reply:
swapped to env var